README.md

```diff
@@ -99,10 +99,14 @@ yargs engine.
 * `argv`: an object representing the parsed value of `args`
   * `key/value`: key value pairs for each argument and their aliases.
   * `_`: an array representing the positional arguments.
+  * [optional] `--`:  an array with arguments after the end-of-options flag `--`.
 * `error`: populated with an error object if an exception occurred during parsing.
 * `aliases`: the inferred list of aliases built by combining lists in `opts.alias`.
-* `newAliases`: any new aliases added via camel-case expansion.
-* `configuration`: the configuration loaded from the `yargs` stanza in package.json.
+* `newAliases`: any new aliases added via camel-case expansion:
+  * `boolean`: `{ fooBar: true }`
+* `defaulted`: any new argument created by `opts.default`, no aliases included.
```

Comment:
@juergba what's your reasoning for not including aliases? @coreyfarrell do you think you can work with this limitation?

Reply:
`Parse.detailed()` also returns `aliases` which includes all aliases, so the information would be redundant.
It's no big effort to add the aliases to `defaulted`, I just need a decision.

Reply:
For my use it would be fine if `defaulted` did not list aliases, as long as setting an alias causes the original key to be excluded from defaulted.  Would you mind adding a test for this?
```js
    it('setting an alias removes associated key from defaulted', function () {
      var parsed = parser.detailed('--foo abc', {
        default: { kaa: 'abc' },
        alias: { foo: 'kaa' }
      })
      parsed.argv.should.deep.equal({ '_': [], kaa: 'abc', foo: 'abc' })
      parsed.defaulted.should.deep.equal({})
    })
```

Sorry I don't have a chance to actually run this test right now but in theory it should work.

Reply:
Done, I added your test.